Sources/Internal/DefaultItemProviders.swift

```diff
@@ -142,5 +142,33 @@ extension CalendarViewContent {
         accessibilityTraits: []),
       viewModel: .init(text: dayText, accessibilityLabel: accessibilityLabel))
   }
-
+  
+  static func defaultMonthFooterItemModelProvider(
```

Comment:
if we make the month footer optional, there's no reason to provide a default item. I also think most people won't need this, or if they do, they'll want something very custom, so providing a default doesn't provide a ton of value.

Reply:
Already en route to removing this default provider

Reply:
Removed